website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -4,36 +4,38 @@ id: "user-tokens"
 pagination_next: "docs/dbt-cloud-apis/service-tokens"
 ---
 
-# User tokens <Lifecycle status="team,enterprise"/>
+# Account-scoped personal access tokens <Lifecycle status="team,enterprise"/>
 
-:::note Announcement
+:::Warning
 
-_User tokens will be deprecated on October 22nd, 2024._
+User API tokens have been deprecated and will no longer work. [Migrate](#migrate-from-user-api-keys-to-personal-access-tokens) to personal access tokens to resume services.
 
-The [user API tokens](/docs/dbt-cloud-apis/user-tokens#user-api-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). If you do not rotate your existing user tokens with PATs by October 22nd, the services using the tokens will encounter errors. 
-
-
-Cloud CLI config files need to be re-downloaded before October 22nd, 2024.
+:::
 
-The current API key is located under **Personal Settings → API Key.** 
+Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using service Tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
 
-Please [contact support](/docs/dbt-support#dbt-cloud-support) with any questions or concerns.
+PATs inherit the permissions of the user that created them. For example, if a developer-licensed user with Project Admin role access to specific projects creates a PAT, the token will get the Project Admin role with access to the same projects as the user. These tokens are also account-specific, so if a user has access to more than one dbt Cloud account with the same email address, they need to create a unique PAT for each one of these accounts. 
 
-:::
+## Create a personal access token
```

Comment:
I recommend we identify this here or in the paragraph below this header.
```suggestion
## Create a personal access token (PAT)
```

Reply:
Won't the () make it difficult to link to? I can't recall if that hinders it